clerk.go

```diff
@@ -277,7 +279,24 @@ func setRequestBody(req *http.Request, params Queryable) error {
 	return nil
 }
 
-// Error response handling
+// Sets the params in the request querystring. Any existing values
+// will be preserved, unless overriden. Keys in the params querystring
+// always win.
+func setRequestQuery(req *http.Request, params Params) {
+	if params == nil {
+		return
+	}
+	q := req.URL.Query()
+	paramsQuery := params.ToQuery()
+	for k, values := range paramsQuery {
+		for _, v := range values {
+			q.Set(k, v)
```

Comment:
❓ `Set` or `Add`? As long as we don't consider a possible future case where we must set a query parameter at the SDK level (I don't think there's one), should be 100% fine though.